### ISSUE TITLE
A list of companies which use Codeception

### ISSUE THREAD

**Issue by DavertMik** [Codeception/Codeception#2740]
As you may know, we are actively developing Codeception 2.2 with some really tasty features to come. In order to motivate us you tell your story, on how you use Codeception.

So yes, that's another attempt to list all our customers in one place.

We have a [wiki page](https://github.com/Codeception/Codeception/wiki/Who-is-using-it%3F) where all companies using Codeception can add themselves to list. However, list was not updated for a while, so I decided to start this issue:

### TL;DR Put a small testimonial with your name and the company you are working for. That will help us a lot ) 

Probably we will add your logo to our site.

**Comment by yupe** [comment]
We are using Codeception for some projects and clients - http://amylabs.ru/
Also we use this in my previous job in http://www.megalabs.ru/ru/
Codeception help us testing tons of old spaghetti code =)

**Comment by javigomez** [comment]
We use codeception at http://redcomponent.com to test our Joomla extensions. Like in redCORE (a free-open framework for developing Joomla extensions https://github.com/redCOMPONENT-COM/redCORE#testing. Is integrated with TravisCI, so all tests run automatically on every pull request.

http://Joomla.org also uses Codeception tests for testing the CMS and specially the Weblinks component. See: https://docs.joomla.org/Testing_Joomla_Extensions_with_Codeception

And I'm aware that https://getgrav.org/ the Flat file CMS just started using Codeception for testing it: https://github.com/getgrav/grav/pull/588

**Comment by sunfreakz** [comment]
SIA [Transact Pro](https://www.transactpro.lv) is a financial institution in [Latvia](https://en.wikipedia.org/wiki/Latvia), We are currently using Codeception to ensure everything is as stable as possible. We write component and acceptance tests for our microservices. Our top 3 built-in modules are REST, Db, and Redis . We also develop our own custom made modules in order to work with Docker and Mountebank, which are the future open source projects.

**Comment by Naktibalda** [comment]
This issue was moved to http://phptest.club/t/a-list-of-companies-which-use-codeception/1202
